.gitignore

```diff
@@ -0,0 +1,5 @@
+'.idea' 
+
+'git push --set-upstream origin insertsort' 
+'sortproject.xml' 
```

Comment:
What is this file and why does it need to be here?

Reply:
done